### DISCUSSION TITLE
[Question] Does this work with all games?

### DISCUSSION THREAD

**Discussion by seniorm0ment** [dmadisetti/steam-tui#25] [Q&A]
Does this override the issue of certain games checking if the game was launched from Steam, and if not attempt to launch the Steam client?
I know steam-cli mentions this as an issue.

**Comment by dmadisetti** [comment]
I haven't tested this with an extensive number of game, but you can launch a headless instance of the steam client by pressing the space bar.

Once this is pressed, we try to launch the game directly through `steamcmd`, which in turn communicates to the steam client.

This seems to work for the games I've tested, but I don't own a ton of games. Try it out and let me know!

**Comment by dmadisetti** [comment]
Moved to a discussion over issue ! See #18 #24

**Comment by dmadisetti** [comment]
Does not seem to work with proton games out of the box. Varying success, see #17 for more deatils
